### PR TITLE
Scrollbar removed

### PR DESCRIPTION
**Pull request by sergeycw**
Hi! I removed scrollbar from the textarea.

### CODE REVIEW

File: components/ReadTime.vue [hunk 1]
```diff
@@ -1,48 +1,59 @@
...
-}
-.read-time {
-  text-align: center;
-}
-</style>
 
+  .scrollbar-hidden {
+    height: 400px;
+    border: none;
+    margin: 0px -300px 0px 15px;
```
Comment (Alex61NN5) [inline components/ReadTime.vue]: Hey There,

According to this website https://www.textfixer.com/tutorials/browser-scrollbar-width.php the default width of a scrollbar is 17px

**Review by Alex61NN5 — CHANGES_REQUESTED** [review]

File: components/ReadTime.vue (hunk 1 above)
Comment (Alex61NN5) [reply components/ReadTime.vue]:
```suggestion
    margin: 0px -300px 0px 17px;
```

Comment (Alex61NN5) [reply components/ReadTime.vue]: image

Comment (sergeycw) [reply components/ReadTime.vue]: Thanks, changed it!

**Pull request merged**